### DISCUSSION TITLE
Game activity - rich pressence

### DISCUSSION THREAD

**Discussion by LepaBrenaa** [maltejur/discord-screenaudio#138] [General]
Does Rich Pressence work in `discord-screenaudio`?
I am unable to see the games that I play as my Discord status. It works on Windows normally.

Is there something I need to enable or it simply isn't implemented in here?
Registered Games section in settings looks like this:
![image](https://user-images.githubusercontent.com/48409293/230882123-7ed75393-543f-4fea-90a0-65c99de59e1b.png)

I use the official AUR package.

**Comment by maltejur** [comment]
Do you have NodeJS installed? If not, try installing it. If yes, how do your logs look like?

**Reply by LepaBrenaa** [reply]
I have NodeJS installed, but I don't know exactly what logs you mean.

If by logs you mean starting discord-screeanaudio via terminal, then here it is:
https://gist.github.com/Smoukus/751778d173c4ebeabc69574be3963d55

Rich Pressence with Spotify works just fine. Don't know if it matters.

**Reply by maltejur** [reply]
Oh wait I did not create a release yet where this feature is included, my bad. A new release should be available in the next days where this will be fixed.

**Reply by LepaBrenaa** [reply]
I updated to the latest version now, but it still doesn't work.
Here are the logs (if this is what you mean by logs):
https://gist.github.com/Smoukus/c7585eb8ef5288a8f708afa1bf0d981e

**Reply by maltejur** [reply]
Yep thats what I mean my logs, thanks. The logs look good so far, arRPC is being started and connects. Is the presence actually not displayed for you or are the settings just missing?

**Reply by LepaBrenaa** [reply]
It's not being displayed for me:
![image](https://user-images.githubusercontent.com/48409293/231236845-198d5fb2-1218-4842-bf8c-a6e9e3e6dfd6.png)

What settings should be enabled?

**Reply by LepaBrenaa** [reply]
@maltejur mind sharing those settings needed? Haven't found anything that I already don't have enabled.
